dali/c_api/c_api.cc

```diff
@@ -203,7 +203,11 @@ unsigned daliGetNumOutput(daliPipelineHandle* pipe_handle) {
 template <typename T>
 static void daliCopyTensorListNToHelper(dali::DeviceWorkspace* ws, void* dst, int n,
                                         device_type_t dst_type) {
-  dali::CopyToExternalTensor(&(ws->Output<T>(n)), dst, (dali::device_type_t)dst_type);
+  dali::CopyToExternalTensor(
+    &(ws->Output<T>(n)),
+    dst,
+    (dali::device_type_t)dst_type,
+    (cudaStream_t) 0);
```

Comment:
IMO the stream should be given as an arg of `daliCopyTensorListNToHelper` and TF should decide what to do about it

Reply:
done